src/app/tutorials/angular2-whatsapp-ionic-cli/3.0.0.ts

```diff
@@ -1,10 +1,10 @@
 const markdownUrlBase = '/manuals/';
 
-export const V2_0_0 = {
+export const V3_0_0 = {
```

Comment:
@darkbasic Please create a new file and revert this file, because we want to preserve V2.